### DISCUSSION TITLE
workflow trigger records restriction off

### DISCUSSION THREAD

**Discussion by JohnMaverick** [YetiForceCompany/YetiForceCRM#15094] [How to]
Execution schedule in workflows contains checkbox "workflow trigger records restriction off"
What is this used for?

Thanks

**Comment by rskrzypczak** [comment]
At the moment, this applies to email reports.
You can send daily, weekly or other reports set according to the schedule to a selected group of users.
It makes more sense with the use of TextParser special functions, such as https://github.com/YetiForceCompany/YetiForceCRM/blob/developer/app/TextParser/UserSSalesProcesses.php#L31, https://github.com/YetiForceCompany/YetiForceCRM/blob/developer/app/TextParser/UserOverdue.php#L27, etc.
